package.json

```diff
@@ -122,8 +122,10 @@
   "dependencies": {
     "@twilio/audioplayer": "1.0.6",
     "@twilio/voice-errors": "1.1.1",
+    "@types/md5": "^2.3.2",
     "backoff": "2.5.0",
     "loglevel": "1.6.7",
+    "md5": "^2.3.0",
```

Comment:
Let's use exact version for consistency. I know we now have a lock file, but we should keep this consistent.

Reply:
What is the size difference?

Reply:
What's the best way to calculate that? Just look at the size of the two new packages?

Reply:
Build the package, check the size of the minified/unminified file and compare it to the previous release (download from tags or npm)

Reply:
`md5` and `@types/md5` in the end adds 10kb; `twilio.min.js` goes from 335kb to 345kb